button-examples/Android/android-button-example.md

```diff
@@ -23,213 +21,387 @@ There are four types of buttons:
 3. [Contained button](#contained-button)
 4. [Toggle button](#toggle-button)
 
-![Example of the four button types](assets/button-types.svg)
+![Button types](assets/button-types.svg)
 
 ## Using buttons
 
-<!-- Update your IDE and your Material Components library to the latest version if you are using Material Components for the first time. -->
-
 Before you can use Material buttons, you need to add a dependency to the Material Components for Android library. For more information, go to the [Getting started](https://github.com/material-components/material-components-android/blob/master/docs/getting-started.md) page.
 
-### Making buttons accessible
```

Comment:
We'll still need accessibility instructions, which should be inherited from the UI button class.

Reply:
Yeah, I left this out because I'm currently not sure of the details. I can check the Android docs and/or ask someone on the Android DevRel team.